api/libproject/devstart.rb

```diff
@@ -2472,6 +2472,7 @@ def deploy_tanagra(cmd_name, args)
   ENV.update(read_db_vars(gcc))
   tanagra_db_username = ENV["WORKBENCH_DB_USER"]
   tanagra_db_password = ENV["WORKBENCH_DB_PASSWORD"]
+  tanagra_db_cloudSqlInstance = ENV["CLOUD_SQL_INSTANCE_NAME"]
```

Comment:
I'm not certain if this env var is populated. I used it because I saw it [here](https://github.com/all-of-us/workbench/blob/8cb2bb535c83058759beb1a73cd7e714d70030db/api/src/main/webapp/WEB-INF/appengine-web.xml.template#L10).

Reply:
Yes, this is located in our vars.env in GCS, which will evaluate to `all-of-us-workbench-test:us-central1:workbenchmaindb`